pkgs/development/python-modules/aiohttp/default.nix

```diff
@@ -58,14 +72,20 @@ buildPythonPackage rec {
 
   disabledTests = [
     # Disable tests that require network access
+    "test_client_session_timeout_zero"
     "test_mark_formdata_as_processed"
+    "test_requote_redirect_url_default"
   ] ++ lib.optionals stdenv.is32bit [
     "test_cookiejar"
   ] ++ lib.optionals stdenv.isDarwin [
     "test_addresses"  # https://github.com/aio-libs/aiohttp/issues/3572, remove >= v4.0.0
     "test_close"
   ];
 
+  disabledTestPaths = [
+    "test_proxy_functional.py" # FIXME package proxy.py
```

Comment:
Is it this https://pypi.org/project/proxy.py/ ?

Reply:
Yes